.fixtures.yml

```diff
@@ -0,0 +1,6 @@
+# This file can be used to install module depdencies for unit testing
+# See https://github.com/puppetlabs/puppetlabs_spec_helper#using-fixtures for details
+---
+fixtures:
+  forge_modules:
+#     stdlib: "puppetlabs/stdlib"
```

Comment:
This code has a dependency on `puppet/provider/parsedfile`, but it doesn't seem like we can express this dependency here